docs/csharp/language-reference/keywords/snippets/Extensions.cs

```diff
@@ -1,10 +1,24 @@
 ﻿namespace keywords;
 
 // <ExtensionMembers>
+/// <summary>
+/// This static class contains extension members for numeric sequences.
+/// </summary>
 public static class NumericSequences
 {
+    /// <summary>
+    /// Extensions for integer sequences.
+    /// </summary>
+    /// <param name="sequence">The sequence used as a receiver.</param>
```

Comment:
If someone was just reading the docs, without regard to where/how the extension member was defined, would they know what "receiver" means?